layouts/shortcodes/EOL_manage.md

```diff
@@ -1,9 +1,13 @@
 <div class="admonition-warning"><p class="admonition-warning-title">Warning</p><div class="admonition-warning-text">
 
-Chef Manage is
-[deprecated](/versions.html#deprecated-products-and-versions) and does
-not work with Chef Automate 2.0. See our
-[documentation](https://automate.chef.io/docs/quickstart/) to learn more
-about Chef Automate 2. This document is no longer maintained.
+Chef Manage is [deprecated](/versions/#deprecated-products-and-versions) and is
+not supported with the Chef Infra Server and Automate 2 deployment method. Chef
+Manage is still supported for non-Automate 2 deployment methods.
```

Comment:
Double-check this, because it sounds like we would actively support a new A1 installation.

Reply:
We could change it to `Manage is still supported with existing non-Automate 2 deployment methods.`